### ISSUE TITLE
Getting the server up

### ISSUE THREAD

**Issue by RodrigoCMoraes** [wdsrocha/dojo#21]
Hi o/

I was unable to get the [server](https://github.com/wdsrocha/dojo/tree/main/packages/server) up by just running `docker-compose up` and execute the proper commands explained in [README.md](https://github.com/wdsrocha/dojo/tree/main/packages/server). 

But after while and a little of web documentation and search I was able to get it up with the configuration below:

```shell
POSTGRES_USER=admin
POSTGRES_PASSWORD=admin
POSTGRES_DB=dev

PGADMIN_DEFAULT_EMAIL=admin@admin.com
PGADMIN_DEFAULT_PASSWORD=admin

JWT_EXPIRATION_TIME=31536000
JWT_SECRET=foobar 

POSTGRES_HOST=0.0.0.0
POSTGRES_PORT=5432

QUEUE_HOST=localhost
QUEUE_PORT=8001

CODEFORCES_CLIENT_USERNAME=my_username
CODEFORCES_CLIENT_PASSWORD=my_password

URI_BOT_EMAIL=my_username
URI_BOT_PASSWORD=my_password
```

basically this code was inside a file in root of [server](https://github.com/wdsrocha/dojo/tree/main/packages/server) named as `dev.env` than I executed the following command:

```shell
export $(cat .env) && yarn start:dev
``` 

> the usernames and passwords here are fake

**Comment by RodrigoCMoraes** [comment]
Please document this step :)

**Comment by wdsrocha** [comment]
Fixed in #24
